src/main/java/dan200/computercraft/ComputerCraft.java

```diff
@@ -277,6 +281,9 @@ public void preInit( FMLPreInitializationEvent event )
         Config.turtlesCanPush = Config.config.get( Configuration.CATEGORY_GENERAL, "turtlesCanPush", turtlesCanPush );
         Config.turtlesCanPush.setComment( "If set to true, Turtles will push entities out of the way instead of stopping if there is space to do so" );
 
+        Config.minTimeBetweenSounds = Config.config.get( Configuration.CATEGORY_GENERAL, "minTimeBetweenSounds", minTimeBetweenSounds);
+        Config.minTimeBetweenSounds.setComment("The minimum time in between calls to sound.play on one computer in milliseconds" );
```

Comment:
You'll need to add a language key to `en_US.lang`. Something like `gui.computercraft:config.min_time_between_sounds=Minimum time between sounds`. Make sure it fits on the screen though. It might also be a good idea to add a `.setMinValue(0)` so people cannot use negatives.